narwhals/functions.py

```diff
@@ -88,15 +96,60 @@ def from_dict(data: dict[str, Any], *, native_namespace: Any) -> DataFrame[Any]:
         └─────┴─────┘
     """
     if native_namespace is get_polars():
-        native_frame = native_namespace.from_dict(data)
+        if schema:
+            from narwhals._polars.utils import (
+                reverse_translate_dtype as polars_reverse_translate_dtype,
```

Comment:
Aliasing is for mypy complaining that pandas one takes different number of arguments